jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -209,6 +247,14 @@ public Instant getCreationTime() {
     return labels;
   }
 
+  public boolean isEntrypointInferredFromBaseImage() {
+    return isEntrypointInferredFromBaseImage;
```

Comment:
I'm also thinking perhaps we should have the field `entrypoint` as `null` to indicate that it should be inherited. The API for `ContainerConfiguration` would be the same, but the builder's setter could be called `setInheritEntrypoint` and it would just set the entrypoint to null. This would make it so that we never have a state where `getEntrypoint` returns something while `shouldInheritEntrypoint` is true.

Reply:
@GoogleContainerTools/java-tools ?

Reply:
I am also leaning toward to not having this extra flag but interpreting the `null` value of `entrypoint` as inheriting it from the base image. And I think we can apply this policy consistently across other fields.

Does it need an additional `setInheritEntrypoint`? The builder's `setEntrypoint` already accepts `null`.

Reply:
And if we go for the null-means-inherit, I think we also need to consider the validity of possible `Entrypoint` and `Cmd` combinations. For example, with `Dockerfile`, if you define a new `Entypoint` but no `Cmd`, `Cmd` is not inherited but set to `null` explicitly in the image metadata. On the other hand, if you define a new `Cmd` but no `Entrypoint`, you inherit `Entrypoint`. So, if `Entrypoint` is non-null, we shouldn't inherit `Cmd` even if `Cmd` is null.